tests/profiles/name_test.py

```diff
@@ -93,12 +93,27 @@ def test_check_monospace():
     ttFont["post"].isFixedPitch = IsFixedWidth.NOT_MONOSPACED
 
     # Now we mark it as monospaced on the OS/2 and it should also fail:
+    original_proportion = ttFont["OS/2"].panose.bProportion
     ttFont["OS/2"].panose.bProportion = PANOSE_Proportion.MONOSPACED
     assert_results_contain(check(ttFont),
                            FAIL, 'bad-panose-proportion',
                            'with a non-monospaced font with bad'
                            ' OS/2.panose.bProportion value (MONOSPACED) ...')
 
+    # restore good value
+    ttFont["OS/2"].panose.bProportion = original_proportion
+
+    # Now we try with very little ASCII characters in the font
+    cmap = ttFont["cmap"]
+    for subtable in list(cmap.tables):
+        # Remove A-Z, a-z from cmap
+        for code in list(range(0x41, 0x5B)) + list(range(0x61, 0x7B)):
```

Comment:
this is perfectly good. Could be more explicit this way, though:
```python
for code in list(range(ord('a'), ord('z')+1)) + list(range(ord('A'), ord('Z')+1)):
```

or even better:

```python
import string
for code in list(map(ord, string.ascii_letters)):
```

Reply:
Nice, importing and using `string` now.